api/src/certification/complementary-certification/domain/models/BadgeToAttach.js

```diff
@@ -19,6 +20,7 @@ class BadgeToAttach {
     this.temporaryCertificateMessage = temporaryCertificateMessage;
     this.stickerUrl = stickerUrl;
     this.createdBy = createdBy;
+    this.minimumEarnedPix = minimumEarnedPix;
```

Comment:
Cf remarque plus bas

```suggestion
    this.minimumEarnedPix = minimumEarnedPix ?? 0;
```